.github/workflows/ScriptTests.yml

```diff
@@ -30,15 +30,15 @@ jobs:
             - name: "Defining environment variables"
               if: startsWith(matrix.os, 'windows')
               run: |
-                echo "CHANGED_PACKAGES_FILES=$('${{ steps.files.outputs.all }}' | Select-String 'packages')" >> $GITHUB_ENV
-                echo "PARTIAL_BUILD=$(If ($CHANGED_PACKAGES_FILES -ne '' -AND '${{ github.event_name }}' -eq 'pull_request'){ echo 'true' } Else { echo 'false' })" >> $GITHUB_ENV
-                echo "SINCE_ARG=$(If ($PARTIAL_BUILD -eq 'true' ){ echo '--since ${{ github.event.pull_request.base.ref }}' } Else { echo '' })" >> $GITHUB_ENV
+                $CHANGED_GLOBAL_FILES = "$('${{ steps.files.outputs.all }}' | Select-String -Pattern '^packages' -NotMatch)"
```

Comment:
Maybe we should extract this 3 lines in a reusable action? Would help avoiding 4x duplication

Reply:
Would be nice.. but they depend on the previous action

Reply:
Indeed we can extract, let’s just merge this one because builds are now stuck because perna is not being partially executed due to wrong values evaluation during declaration of vars..

Reply:
And then extract in an action combining both cases :)